draft-ietf-quic-recovery.md

```diff
@@ -989,6 +988,13 @@ The recovery period limits congestion window reduction to once per round trip.
 During recovery, the congestion window remains unchanged irrespective of new
 losses or increases in the ECN-CE counter.
 
+## Application Limited Sending
+
+If the sender is sufficiently application limited that the congestion window is
+not fully utilized, the congestion window should not be increased in slow start
+or congestion avoidance.  Senders should consider themselves application limited
+if bytes in flight when receiving an ACK frame are more than a max datgram size
+less than the congestion window.
```

Comment:
This last sentence is a run-on and unnecessarily detailed. I would simply say something like "if the congestion window is not fully utilized." The detail of kMaxDatagramSize is a rounding thing, which is probably too much detail in this descriptive text.

Reply:
In that case, I think I should just remove that sentence?